sites/akvo-flow-web/src/util/Languages.js

```diff
@@ -742,6 +742,10 @@ export const Locale = {
         en: "Confirm",
         de: "Bestätigen"
     },
+    overviews: {
+        en: "Overview and Download",
+        de: "Überblick und Download"
```

Comment:
Does *Download* needs to be translated?

Reply:
e.g. *Herunterladen*

Reply:
Partner has specified this translation 🤔 as you can see in the related issue https://github.com/akvo/akvo-tech-consultancy/issues/497

Reply:
Then, I have said nothing 🤐